src/libraries/System.Security.Cryptography/src/System/Security/Cryptography/OpenSsl.NotSupported.cs

```diff
@@ -279,6 +279,14 @@ public SafeEvpPKeyHandle(IntPtr handle, bool ownsHandle) : base(handle, ownsHand
         public static SafeEvpPKeyHandle OpenPublicKeyFromEngine(string engineName, string keyId) =>
             throw new PlatformNotSupportedException(SR.PlatformNotSupported_CryptographyOpenSSL);
 
+        [UnsupportedOSPlatform("android")]
+        [UnsupportedOSPlatform("browser")]
+        [UnsupportedOSPlatform("ios")]
+        [UnsupportedOSPlatform("tvos")]
+        [UnsupportedOSPlatform("windows")]
+        public static SafeEvpPKeyHandle OpenKeyFromProvider(string providerName, string keyUri) =>
+            throw new PlatformNotSupportedException(SR.PlatformNotSupported_CryptographyOpenSSL);
```

Comment:
Nothing to change here, but sharing some words.

I hate that we have to keep these attributes in sync across multiple files.

The good news is that, aside from direct interop (which, OK, this is), we're not putting platform support attrs on crypto any more.

In the future (not this change) I think we'll solve it via "required partials".  That is, classes that are inherently built differently across different platforms will have the "API presentation" file (xmldoc, attributes), and then the platform-impl file.  It also solves the question of where do we write xmldoc and how many copies have to match. https://github.com/dotnet/runtime/blob/main/src/libraries/Common/src/System/Security/Cryptography/X509Certificates/X509CertificateLoader.cs was me prototyping that concept; though I didn't have to double check that attributes are unioned across the two... which would be required for solving _this_ problem.